packages/console/scripts/generate-jwt-customizer-type-definition.ts

```diff
@@ -43,6 +45,11 @@ const createJwtCustomizerTypeDefinitions = async () => {
     'JwtCustomizerUserContext'
   );
 
+  const jwtCustomizerGrantContextTypeDefinition = inferTsDefinitionFromZod(
```

Comment:
i know it's following the convention here. however, it looks strange to have "ContextType" here but use it as "data" in the below files. maybe we should rename variables here.